src/pages/home/home.scss

```diff
@@ -0,0 +1,73 @@
+.homeContainer {
+  height: 100%;
+
+  & .mainHeaderBackground {
```

Comment:
Please remove the nesting, we don't really need to nest classes, this is because we are using css modules.

Reply:
For consistency sake, are we going with camel case style names? If so, let's make sure this is the consistent across all scss files.

Reply:
I personally like `.mainHeaderBackground ` as oppose of `.MainHeaderBackground`, but that's just my personal preference, I'm open to any of those.

I do agree that we need to be consistent across our codebase. @arecvlohe can you please document in our FE google doc which on to use?

Reply:
@arecvlohe , @crysfel I have followed snake case classname as per review comments in #67

Reply:
So now we have snake case and camel case. I suppose camel case is more ergonomic. I initially wanted to go with snake case because that PR, #67, had a mix of BEM snake case and camel case. We didn't really need the BEM part of it but the snake case hung around. It was a mistake on my part to overlook this. I think now going with camel case is more ideal. Less need to reach for the _ when typing. Thoughts?